server/models/db/school.js

```diff
@@ -0,0 +1,58 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const UserTypes = require('../../constants/userTypes.json');
+const Tables = require('../../constants/tables.json');
+
+const WorkerDetails = require('../data/Worker');
+
+// This method returns the corresponding workers given school ID. If the school ID is incorrect, it will return null. 
+// If either the tableName or Object are null or undefined, an error will be thrown.
+async function getWorkerIdsForSchool(schoolId) {
+    if (_.isNil(schoolId)) return null;
+    try {
+        //Query to recieve worker IDs from Service Worker School Table 
+        const workerIds = await db.any(`select worker_id from Service_Worker_School where school_id='${schoolId}';`);
+        //return workerIds;
+
+        var queryParams = `${Tables.worker}.worker_id, ${Tables.worker}.first_name, ${Tables.worker}.last_name`;
+        var tableJoins = ` ${Tables.worker} `;
+        //queryCond += ` and ${Tables.appointment}.student_id=${studentId}`;
+        //workerIds = 8000000;
+        const workerIds_2 = JSON.stringify(workerIds);
```

Comment:
Hi Simran,
I would recommend leveraging the .map function in lodash to convert the current sql response to an array of workerIds. They have a good example in their documentation located: https://lodash.com/docs/#map

Reply:
Hi Amy! 

Thanks a lot! I've reviewed the link you have sent me and implemented the logic in the second query statement.